test/position-managers/Execute.t.sol

```diff
@@ -0,0 +1,146 @@
+
+        (key, poolId) = initPool(currency0, currency1, IHooks(address(0)), 3000, SQRT_PRICE_1_1, ZERO_BYTES);
+
+        // Requires currency0 and currency1 to be set in base Deployers contract.
+        deployAndApprovePosm(manager);
+
+        // Give tokens to Alice and Bob.
+        seedBalance(alice);
+        seedBalance(bob);
+
+        // Approve posm for Alice and bob.
+        approvePosmFor(alice);
+        approvePosmFor(bob);
+
+        // define a reusable range
+        range = LiquidityRange({poolKey: key, tickLower: -300, tickUpper: 300});
+    }
+
+    function test_fuzz_execute_increaseLiquidity_once(uint256 initialLiquidity, uint256 liquidityToAdd) public {
+        initialLiquidity = bound(initialLiquidity, 1e18, 1000e18);
+        liquidityToAdd = bound(liquidityToAdd, 1e18, 1000e18);
+        mint(range, initialLiquidity, address(this), ZERO_BYTES);
+        uint256 tokenId = lpm.nextTokenId() - 1;
+
+        increaseLiquidity(tokenId, liquidityToAdd, ZERO_BYTES);
+
+        bytes32 positionId =
+            keccak256(abi.encodePacked(address(lpm), range.tickLower, range.tickUpper, bytes32(tokenId)));
+        (uint256 liquidity,,) = manager.getPositionInfo(range.poolKey.toId(), positionId);
+
+        assertEq(liquidity, initialLiquidity + liquidityToAdd);
+    }
+
+    function test_fuzz_execute_increaseLiquidity_twice(
+        uint256 initialiLiquidity,
+        uint256 liquidityToAdd,
+        uint256 liquidityToAdd2
+    ) public {
+        initialiLiquidity = bound(initialiLiquidity, 1e18, 1000e18);
+        liquidityToAdd = bound(liquidityToAdd, 1e18, 1000e18);
+        liquidityToAdd2 = bound(liquidityToAdd2, 1e18, 1000e18);
+        mint(range, initialiLiquidity, address(this), ZERO_BYTES);
+        uint256 tokenId = lpm.nextTokenId() - 1;
+
+        Planner.Plan memory planner = Planner.init();
+
+        planner = planner.add(Actions.INCREASE, abi.encode(tokenId, liquidityToAdd, ZERO_BYTES));
+        planner = planner.add(Actions.INCREASE, abi.encode(tokenId, liquidityToAdd2, ZERO_BYTES));
+
+        bytes memory calls = planner.finalize(range.poolKey);
+        lpm.modifyLiquidities(calls, _deadline);
+
+        bytes32 positionId =
+            keccak256(abi.encodePacked(address(lpm), range.tickLower, range.tickUpper, bytes32(tokenId)));
+        (uint256 liquidity,,) = manager.getPositionInfo(range.poolKey.toId(), positionId);
+
+        assertEq(liquidity, initialiLiquidity + liquidityToAdd + liquidityToAdd2);
+    }
+
+    // this case doesnt make sense in real world usage, so it doesnt have a cool name. but its a good test case
+    function test_fuzz_execute_mintAndIncrease(uint256 initialLiquidity, uint256 liquidityToAdd) public {
+        initialLiquidity = bound(initialLiquidity, 1e18, 1000e18);
+        liquidityToAdd = bound(liquidityToAdd, 1e18, 1000e18);
+
+        uint256 tokenId = 1; // assume that the .mint() produces tokenId=1, to be used in increaseLiquidity
```

Comment:
nit: maybe it makes sense to use `nextTokenId` here in case we opt in for a shared setup in the future that also already mints default positions

Reply:
resolved!